### PR TITLE
Adds Home-2 Scenario 

### PR DESCRIPTION
**Pull request by codingblazer**
Adds the Home-2 scenario, the consent scenario.

### CODE REVIEW

**Comment by aanchal07** [comment]
This screen appears on clicking the house for the first time:
image

Why is that so?

**Comment by codingblazer** [comment]
@aanchal07 It was working fine on my device. But after clearing the data again, I am also having the same problem as you. I am looking into this.

**Comment by codingblazer** [comment]
@aanchal07 Actually, there were some references to ScenarioChooser while matching it to database. I have made the changes now.

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java [hunk 1]
```diff
@@ -20,8 +21,10 @@
         public void onClick(View v) {
             Button scenarioChooser = (Button) v;
             if (getmDbHandler().setSessionId(scenarioChooser.getText().toString())) {
+                Log.e("sachin",scenarioChooser.getText().toString());
```
Comment (aanchal07) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: Remove this

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java [hunk 2]
```diff
@@ -20,8 +21,10 @@
...
                 startActivityForResult(new Intent(MapActivity.this, GameActivity.class), 0);
             } else {
+                Log.e("sachin",scenarioChooser.getText().toString());
```
Comment (aanchal07) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: remove this

**Comment by codingblazer** [comment]
@aanchal07 Done

**Pull request merged**